apps/box/src/components/Version.tsx

```diff
@@ -0,0 +1,17 @@
+import { FxText, FxTextProps } from '@functionland/component-library';
```

Comment:
This feels very specific to the `Settings` screen and not sure if it's reusable on other screens/components. You think it's worth moving to a different location?

Reply:
Yeah I did consider putting it in settings but thought it might be used outside of setting say on a welcome screen or about screen.  Happy to move it if you would prefer 😄

Reply:
I'm okay either way on this one, your call!

Reply:
Will push as is for now